portfolio/src/main/webapp/index.html

```diff
@@ -163,19 +164,29 @@ <h3>Swedish</h3>
           <button class="hyperlinked-button buttonGray">GitHub</button></a>
         </div>
       </section>
-      <div class="center-content commentSection">
+      <div id="commentArea" class="center-content commentSection">
         <h2><b>Comment Section</b></h2>
         <form action="/data" method="POST">
-          <h4 class="label">First name:</label>
-          <input type="text" name="userFirstName"><br>
-          <h4 class="label">Last name:</label>
-          <input type="text" name="userLastName">
-          <h4 class="label">Please leave a comment:</h4>
+        <!-- Ask user how many comments they want to see -->
+          <p>How many comments do you want to see?</p>
+          <select name="commentCount" id="commentCount" onchange="getComments()">
+              <option value="1">1</option>
+              <option value="10">10</option>
+              <option value="25">25</option>
+              <option value="50">50</option>
+              <option value="-1">All</option>
+          </select>
+          <br></br>
```

Comment:
The br tag is just `<br>` (there's no need to close it)
https://www.w3schools.com/tags/tag_br.asp

Reply:
Thanks for the link!